el-GR.json

```diff
@@ -1008,20 +1008,20 @@
 	"STREMIO_TV_STREAMS_STILL_LOADING": "Το ${1} ακόμα φορτώνει ...",
 	"STREMIO_TV_PLAYER_SETTINGS": "Ρυθμίσεις προγράμματος αναπαραγωγής",
 	"STREMIO_TV_PLAYER_SUBTITLES_DELAY": "Καθυστέρηση υποτίτλων",
-	"STREMIO_TV_PLAYER_AUDIO_DELAY": "Audio delay",
-	"STREMIO_TV_PLAYER_VIDEO_SCALING": "Video scaling",
+	"STREMIO_TV_PLAYER_AUDIO_DELAY": "Καθυστέρηση ήχου,
```

Comment:
there is a missing double quote before the comma at the end.

Reply:
Sorry, I'll fix that.

Reply:
I fixed it but it isn't shown.